modules/standard/IO.chpl

```diff
@@ -6772,7 +6814,7 @@ proc fileReader.readAll(ref b: bytes): int throws {
 }
 
 /*
-  Read the remaining contents of the fileReader into a an array of bytes.
+  Read the remaining contents of the ``fileReader`` into a an array of bytes.
```

Comment:
Not your fault, but it should just be "an array of bytes" not "a an array of bytes"